elastic/kibana
# [Security Solution] Remove flyout context in url when in rule creation workflow

## Summary

When a user is in rule creation page, after expanding alert flyout for a preview alert:
- Reloading tab should not reopen the flyout - because the information would have been stale
- Clicking back button should not reopen the flyout

https://github.com/elastic/kibana/assets/18648970/d96e1296-01bf-4c50-9f77-a38fe53bc62e


**How to test**
- Enable feature flag `expandableFlyoutInCreateRuleEnabled`
- Generate some alert/event data
- Go to Rules page -> Detection rules -> Create new rule
- Put in some data (like a query), under Rule Preview, expand a row
- Reload page, notice the flyout does not reopen

### Checklist
